lighthouse-core/report/v2/renderer/util.js

```diff
@@ -60,6 +61,16 @@ class Util {
   }
 
   /**
+   * @param {number} ms
+   * @param {number=} granularity Controls how coarse the displayed value is, defaults to 10
+   * @return {string}
+   */
+  static formatMilliseconds(ms, granularity = 10) {
+    const coarseTime = Math.round(ms / granularity) * granularity;
```

Comment:
could replace this guy with: {maximumFractionDigits: 1 }
but..........

afaik that's not supported in node's impl. :)
can you leave a // COMPAT comment here? in a year we could migrate to it.